### ISSUE TITLE
no normalizaton of image data

### ISSUE THREAD

**Issue by justdoGIT** [dhanushkamath/CIFAR10_Simple#1]
I came across your cifar-10 example. It is really good for understanding transfer learning concepts but I have a query, why haven't you normalized your image array. Would this not lead to any problems.

**Comment by dhanushkamath** [comment]
Hi Kamal,

Good catch there.
I believe normalisation is not required here as the data is in the range of
0 to 255. Mean subtraction can probably help. You can check out batch
normalisation too as I've heard it's the most effective approach currently.
Do share the results with me if you've implemented these as I would love to
see the differences.

Sincere regards,
Dhanush Narayan Kamath,
B.Tech-IV,
Electrical Engineering Department,
NIT Surat

**Comment by justdoGIT** [comment]
so that line `x = BatchNormalization()(x)` is doing the job of normalizing the images. is that true

**Comment by dhanushkamath** [comment]
Batch normalization is basically extending the idea of normalization to the
hidden layers for normalizing their activations or outputs. It provides a
considerable boost in training time and accuracy owing to faster and more
precise convergence.

Refer this article for more details -

https://towardsdatascience.com/batch-normalization-in-neural-networks-1ac91516821c

Sincere regards,
Dhanush Narayan Kamath,
B.Tech-IV,
Electrical Engineering Department,
NIT Surat

**Comment by justdoGIT** [comment]
Can you please tell me how to test the data using your model. I am a bit confused whether to apply normalization to the test images and how to run model .predict() from your concatenated model.

**Comment by justdoGIT** [comment]
The problem I am facing is that input of the concat model is 2D while the shape of my test array is (256,256,3). so model.predict is throwing an error. If you have any idea how to test data using the concatenated model, that will be really helpful
